contents/blog/startup-ops-toolkit.md

```diff
@@ -13,63 +13,66 @@ Ok, so you’ve read *Zero to One* and *The Hard Thing About Hard Things*, but w
 
-If you need an Employer of Record service, Deel now offer this too - super useful if you are hiring permanent employees outside the US in a country where you don’t have a legal presence.
+> Time suck alert - you will need to register in the relevant state _before_ you can hire someone and onboard them onto Gusto. Online state registration laws vary and all of their websites are garbage. <a target="_blank" rel="noopener noreferrer nofollow" href="https://agent.middesk.com/">Middesk</a> can handle this for you on a pay as you go basis and will save you 3+ hour each time. If you want to go further, we use their new $2k/quarter unlimited state filing and reporting service which is a huge time saver. You will forget a state's tax filing deadline and get fined - better to outsource the problem. 
 
-### Contracts
+### Contractors (Global)
 
-Orrick have a <a target="_blank" rel="noopener noreferrer nofollow" href="https://www.orrick.com/en/Total-Access/Tool-Kit/Start-Up-Forms/Employment-and-Consultant">handy free set of templates</a> you can use! (US only)
+<a target="_blank" rel="noopener noreferrer nofollow" href="https://www.letsdeel.com/">Deel</a> is the way to go. They cover pretty much any country and currency you’ll need, handle all the compliance stuff for you, and generally make the process way more pleasant than it has any right to be. People complain that their support has worsened as they've experienced massive growth, but I still find them very responsive and helpful. It’s one of my favorite ops products. Deel charge $35/month per contractor, and they also handle monthly payroll. 
 
-You need both an Offer Letter and a Confidential Information and Invention Assignment Agreement (US only). The CIIAA means that your employees agree that their work is confidential and whatever they build at work is owned by the company. Not having this = major problems in future, especially for fundraising.
+### Permanent employees (non-US)
 
-Deel have free templates integrated into their platform already, so you’ll only usually use these for permanent employees, unless you want to use a customized contractor agreement in Deel.
+Assuming you don't have a legal entity in every country, you'll need to use an Employer of Record service to hire permanent employees internationally. Deel also offer this service - it might _feel_ steep at $599/month per employee, but it's honestly worth it (and they undercut other providers last time I checked). The process is pretty straightforward and self-serve. Couple of things to note:
```

Comment:
```suggestion
Assuming you don't have a legal entity in every country, you'll need to use an Employer of Record service to hire permanent employees internationally. Deel also offers this service – it might _feel_ steep at $599/month per employee, but it's honestly worth it (and it undercuts other providers last time I checked). The process is pretty straightforward and self-serve, but here are a few things to note:
```